<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tektronix_send_cmd_dialog</class>
 <widget class="QDialog" name="tektronix_send_cmd_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отправить команду</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLineEdit" name="cmd_edit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Описание команды</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="idn_button">
       <property name="text">
        <string>*IDN?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="error_buttons">
       <property name="text">
        <string>Ошибки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="read_specter_button">
       <property name="text">
        <string>Спектр</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="cmd_description_text_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="send_cmd_button">
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Команда</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
